spec/jobs/reports/drop_off_report_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Reports::DropOffReport do
-  let(:report_date) { Date.new(2023, 12, 12).in_time_zone('UTC') }
+  let(:report_date) { Date.new(2023, 12, 12).in_time_zone }
```

Comment:
What does `in_time_zone` do if not passed an argument?

Reply:
It uses the machine's time zone. An earlier version I wrote had setup variables here that used `Time.zone.yesterday` which is also uses the machine's time zone. The test failed unless both were using the same time zone because the time zone changes what `to_s` does when formatting the report.

I've since moved that test setup to cloudwatch_client_spec.rb. I should revert this line